Libraries/CMSIS/Device/Maxim/MAX32680/Include/afe_adc_zero_regs.h

```diff
@@ -563,7 +563,7 @@ extern "C" {
  * @{
  */
 #define MXC_F_AFE_ADC_ZERO_PART_ID_REV_ID_POS          0 /**< PART_ID_REV_ID Position */
-#define MXC_F_AFE_ADC_ZERO_PART_ID_REV_ID              ((uint32_t)(0x7UL << MXC_F_AFE_ADC_ZERO_PART_ID_REV_ID_POS)) /**< PART_ID_REV_ID Mask */
+#define MXC_F_AFE_ADC_ZERO_PART_ID_REV_ID              ((uint32_t)(0x3FUL << MXC_F_AFE_ADC_ZERO_PART_ID_REV_ID_POS)) /**< PART_ID_REV_ID Mask */
```

Comment:
I don't think this looks right.  Its only changed for 680 and only in this header file.  The svd is not updated to match either, so this is likely a mistake.

Reply:
Nice catch, fixed the SVD and updated the ME16's PART_ID register. However, I think we're dropping support for the ME20, so I will leave the ME20 files as there are to match the UG.